ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -143,8 +142,7 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
-- always_run: false
-  as: e2e-aws-gatewayapi-conformance
+- as: e2e-aws-gatewayapi-conformance
```

Comment:
@alebedev87 would you mind updating L135 to TechPreviewNoUpgrade and/or removing the FEATURE_SET/FEATURE_GATES env variables from here when the time is right?

Reply:
I removed `e2e-aws-gatewayapi` job because `TestGatewayAPI` is now part of the standard test suite (`e2e-*-operator` jobs). `e2e-aws-gatewayapi-conformance` is updated too: `TechPreviewNoUpgrade` featureste is removed.